src/Strategy/DateTimeFormatterStrategy.php

```diff
@@ -48,7 +48,8 @@ public function __construct($format = DateTime::RFC3339, DateTimeZone $timezone
     public function extract($value)
     {
         if ($value instanceof DateTime) {
```

Comment:
Why we don't accept `DateTimeInterface` @weierophinney?

Reply:
@malukenho We do as of #70. ;-)